README.md

```diff
@@ -1,40 +1,15 @@
-# Create a TODO app with Squid
+# Create a to-do app with Squid
 
+Welcome to Squid Cloud! This project contains a to-do application in which you will add Squid functionality. The steps to complete the project are displayed in the frontend, so the first step is to get the frontend up and running locally.
 
-1. In the [Squid Cloud Console](https://console.squid.cloud) create a new app called `todo`.
 
-2. Connect the Squid backend to the `todo` by scrolling in the console **Overview** page to the **Backend project** section and selecting **Create .env file**. Copy the command.
-
-3. Open a terminal window and change to the `backend` directory.
-
-```bash
-cd todo/backend
-```
-
-4. Install the required packages:
+1. The frontend of this project is a React app that uses [Vite](https://vitejs.dev/guide/cli.html). Note that Squid works with any TypeScript client; this project just happens to use React with Vite. Intall the Vite CLI:
 
 ```bash
-npm install
+npm install -g vite
```

Comment:
@jenperson you shouldn't need to do this step, `vite` should already be installed if you run `npm install` in the `frontend` directory.